src/shims/posix/fs.rs

```diff
@@ -251,22 +251,70 @@ impl FileDescriptor for io::Stderr {
     }
 }
 
+#[derive(Debug)]
+struct DevNull;
+
+impl FileDescriptor for DevNull {
+    fn as_file_handle<'tcx>(&self) -> InterpResult<'tcx, &FileHandle> {
+        throw_unsup_format!("/dev/null cannot be used as FileHandle");
```

Comment:
Talking about `/dev/null` here is very confusing since this has nothing to do with that file